composer.json

```diff
@@ -107,5 +108,11 @@
             "allow-contrib": false,
             "require": "4.3.*"
         }
+    },
+    "repositories": {
+        "itk-dev/user-bundle": {
+            "type": "vcs",
+            "url": "https://github.com/itk-dev/user-bundle"
+        }
```

Comment:
I've added `itk-dev/user-bundle` to packagist.org: https://packagist.org/packages/itk-dev/user-bundle.

On a side note: We need to discuss how to handle https://github.com/itk-dev/user-bundle/ and https://github.com/itk-dev/user-management-bundle (at least the latter should depend on the former).